src/printer.js

```diff
@@ -1183,19 +1183,27 @@ function genericPrintNoParens(path, options, print) {
         path.call(print, "identifier")
       ]);
     case "ClassBody":
-      if (n.body.length === 0) {
+      if (!n.comments && n.body.length === 0) {
         return "{}";
```

Comment:
See other place where we do dangling comments. They just add them here: `concat(["{", printDanglingComments(node), "}"])` or something like that (just wrote off the top of my head, probably not correct)